src/state_point.cpp

```diff
@@ -93,6 +93,8 @@ openmc_statepoint_write(const char* filename, bool* write_source)
     write_attribute(file_id, "photon_transport", settings::photon_transport);
     write_dataset(file_id, "n_particles", settings::n_particles);
     write_dataset(file_id, "n_batches", settings::n_batches);
+    write_dataset(file_id, "n_max_lost_particles", settings::n_max_lost_particles);
+    write_dataset(file_id, "relative_max_lost_particles", settings::relative_max_lost_particles);
```

Comment:
I don't see a compelling reason to put these in the statepoint file. There are many other settings that we don't store in the statepoint. The purpose is really to save information from a simulation as opposed to what settings were in use.

Reply:
Ah good to know, I was a little uncertain if they should be output by the statepoint.